content/docs/install/completion.md

```diff
@@ -134,7 +134,7 @@ $ exec $SHELL -l
 This step is optional but will make the DVC output look much nicer, by adding
 more colors to it. Add the following to your `~/.zshrc`:
 
-```zsh
+```bash
```

Comment:
why this change? it is zsh, right?

Reply:
Basically because both `hljs` and `prsim` actually didn't have syntaxes for `zsh` or `sh` and they fallback for default mode then you use it.

See: https://github.com/conorhastings/react-syntax-highlighter/tree/master/src/languages/hljs
And https://github.com/conorhastings/react-syntax-highlighter/tree/master/src/languages/prism

And we also are not importing them it https://github.com/iterative/dvc.org/blob/master/src/components/Documentation/Markdown/index.js either.

Default mode for `hljs` here is something c-like and is highlight some things as a result. But `prismjs` that `gatsby` uses just leave code unhighlighted.  So here I have two options – either write `zsh` syntax from scratch (but we only use it in this one place) or try to find something close enough to the default `hljs` syntax. I choose later for now.

Reply:
sounds good, @iAdramelk ! thanks.